html-relative-links/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Home Page</title>
+    <style>
+      img {
+        width: 300px;
+      }
+    </style>
+  </head>
+  <body>
+    <ul>
+      <li>Home Page</li>
+      <li>
+        <a href="../html-relative-links/about-me/about-me.html">About Me</a>
+      </li>
+      <li>
+        <a href="../html-relative-links/projects/projects.html">Projects</a>
+      </li>
+      <li>
+        <a href="../html-relative-links/contact-us/contact-us.html"
+          >Contact Us</a
+        >
+      </li>
+    </ul>
+    <p>Hello Homepage!</p>
+    <img
+      src="../html-relative-links/images/javascript.png"
```

Comment:
We want to take the shortest distance possible, so drop the `../html-relative-links`. It should just be `about-me/about-me.html` for example